modules/core/arrow-test/src/main/kotlin/arrow/test/laws/BracketLaws.kt

```diff
@@ -0,0 +1,92 @@
+package arrow.test.laws
+
+import arrow.Kind
+import arrow.core.Either
+import arrow.effects.typeclasses.Bracket
+import arrow.effects.typeclasses.ExitCase
+import arrow.test.generators.genConstructor
+import arrow.test.generators.genFunctionAToB
+import arrow.test.generators.genThrowable
+import arrow.typeclasses.Eq
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+
+object BracketLaws {
+  inline fun <F> laws(BF: Bracket<F, Throwable>,
+                      noinline cf: (Int) -> Kind<F, Int>,
```

Comment:
You don't need this parameter, as it's `just` from `Applicative`.

Reply:
Alright, removed now!